frontend-service/src/test/java/com/amazonaws/blox/frontend/operations/EnvironmentControllerTestCase.java

```diff
@@ -51,13 +60,29 @@
   protected static final String ATTRIBUTE_NAME = "TestAttributeName";
   protected static final String ATTRIBUTE_VALUE = "TestAttributeValue";
   protected static final String ACCOUNT_ID = "1234567890";
-  protected MockHttpServletRequest servletRequest;
-  ApiGatewayRequestContext requestContext = new ApiGatewayRequestContext();
-  DataService dataService = mock(DataService.class);
+  @Autowired DataService dataService;
+  @Autowired HttpServletRequest servletRequest;
+
+  @Configuration
+  @Import(MapperConfiguration.class)
+  @ComponentScan("com.amazonaws.blox.frontend.operations")
```

Comment:
Hm. Is there a reason that this is setting up a totally new config, rather than importing the main Application config and overriding the necessary beans?

Reply:
The Spring runner cannot work with both `@ContextConfiguration` and `@ComponentScan`. So need to use a Configuration file to load all configurations. 

As for directly using a main Application config.That one actually includes too many dependencies,  and have system properties for lambda configuration, enable mvc etc.   The `EnvironmentControllerTestCase` is still lightweight unit test, so use a new config just for controller dependencies.